src/Renderer/GiftCardRendererInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+namespace Setono\SyliusGiftCardPlugin\Renderer;
+
+use Setono\SyliusGiftCardPlugin\Model\GiftCardConfigurationInterface;
+use Setono\SyliusGiftCardPlugin\Model\GiftCardInterface;
+
+interface GiftCardRendererInterface
+{
+    /**
+     * A boolean indicating whether the rendered PDF will be base64 encoded or not
+     *
+     * Default: false
+     */
+    public const OPTION_BASE_64 = 'base64';
```

Comment:
Is that really a boolean ? I mean, it looks like a string right now :thinking: :laughing:

Reply:
Didn't know how to document it :D The option when sent to the `GiftCardRendererInterface` is a boolean :D